web/wwwroot/css/welcome.css

```
@@ -0,0 +1,593 @@
+    -moz-osx-font-smoothing: grayscale;
+}
+
+/* Screen-reader-only utility — matches VIPER's _VIPERLayout pattern. */
+.sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)